src/main/java/mate/academy/springboot/swagger/util/SortByParser.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springboot.swagger.util;
+
+import org.springframework.data.domain.Sort;
+
+public interface SortByParser {
```

Comment:
```suggestion
public interface SortingOptionsParser {
```
or you could make just util class with static method